docs/docs/development/setup.md

```diff
@@ -513,9 +520,20 @@ On Windows, only two environment variables need to be set for ZMK to build prope
 
 #### For Zephyr
 
-By default, the Zephyr™ SDK will create a file named `~/.zephyrrc` with the correct environment variables to build ZMK.
+By default, the Zephyr™ SDK _should_ create a file named `~/.zephyrrc` with the correct environment variables to build ZMK.
 We suggest two main [options](https://docs.zephyrproject.org/2.5.0/guides/env_vars.html#option-3-using-zephyrrc-files) for how to load those settings.
 
+:::note `.zephyrrc` on macOS
+
+In case you don't have `~/.zephyrrc`, these are the two environment variables you will need:
+
+```
+export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+export GNUARMEMB_TOOLCHAIN_PATH=$(brew --prefix)
+```
+
+:::
+
```

Comment:
```suggestion
```

This steps is no longer needed, as the instructions are updated to add these lines to the .zephyrrc.

Reply:
> This steps is no longer needed, as the instructions are updated to add these lines to the .zephyrrc.

Hey @rtitmuss, I can’t see where in the instructions these lines get added to the `.zephyrrc` file. Please could you clarify?

Because [another user reported a missing `~/.zephyrrc` file](https://github.com/zmkfirmware/zmk/pull/917/files#r741073824), I wondered if this section should remain, but tweaked to be more of a generic “here’s how to fix a missing `~/.zephyrrc` file”.